sdk/servicebus/azure-servicebus/samples/async_samples/sample_code_servicebus_async.py

```diff
@@ -57,7 +57,7 @@ async def example_create_servicebus_sender_async():
     from azure.servicebus.aio import ServiceBusSender
     servicebus_connection_str = os.environ['SERVICE_BUS_CONNECTION_STR']
     queue_name = os.environ['SERVICE_BUS_QUEUE_NAME']
-    queue_sender = ServiceBusSender.from_connection_string(
+    queue_sender = ServiceBusSender._from_connection_string(
```

Comment:
aha, should we just remove the these code along with the snippets docstring on `Sender/Receiver.from_connection_string`? otherwise they would show up in our api references

Reply:
The docstrings show up even with "_"?  Ugh.  I'm torn because those are really useful docs and I'd like to have them internal since we put the effort in to make them, but if needed we can.

Reply:
you're right, I generated the doc it didn't show up :P